pkg/logging/kinesis/root.go

```diff
@@ -0,0 +1,28 @@
+package kinesis
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/config"
+)
+
+// RootCommand is the parent command for all subcommands in this package.
+// It should be installed under the primary root command.
+type RootCommand struct {
+	common.Base
+	// no flags
+}
+
+// NewRootCommand returns a new command registered in the parent.
+func NewRootCommand(parent common.Registerer, globals *config.Data) *RootCommand {
+	var c RootCommand
+	c.Globals = globals
+	c.CmdClause = parent.Command("kinesis", "Manipulate Fastly service version Kinesis logging endpoints")
```

Comment:
I'm not a fan of the wording used in the CLI currently as it doesn't read like a correct sentence.

```suggestion
	c.CmdClause = parent.Command("kinesis", "Manipulate a Kinesis logging endpoint for a specific Fastly service version")
```

This could be something I fix more broadly in a separate PR though.

Oh, actually -- is this command even reachable (after looking at `Exec`)?

Reply:
> Oh, actually -- is this command even reachable (after looking at Exec)?

All command groups have unreachable roots in the tree. This is the root for this group of commands, the text is only used in `help` output when, for instance, `fastly logging --help` is ran.